packages/fast-components-react-base/src/text-field/text-field.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, { HandledProps } from "../foundation";
+import {
+    ITextFieldHandledProps,
+    ITextFieldManagedClasses,
+    ITextFieldUnhandledProps,
+    TextFieldType
+} from "./text-field.props";
+import { IManagedClasses, ITextFieldClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { get } from "lodash-es";
+
+/* tslint:disable-next-line */
+class TextField extends Foundation<ITextFieldHandledProps & IManagedClasses<ITextFieldClassNameContract>, React.HTMLAttributes<HTMLInputElement>, {}> {
+    protected handledProps: HandledProps<ITextFieldHandledProps & IManagedClasses<ITextFieldClassNameContract>> = {
+        disabled: void 0,
+        error: void 0,
+        placeholder: void 0,
+        managedClasses: void 0,
+        type: void 0,
+        value: void 0
```

Comment:
value won't be passed if we are handling it but not applying it to the element.

If we don't plan on doing anything with it, let's remove this from here and props to ensure users can pass it as expected.